packages/forms-web-app/src/dynamic-forms/controller.js

```diff
@@ -122,8 +122,11 @@ exports.list = async (req, res, pageCaption, viewData) => {
 		summaryListData.sections.push(sectionView);
 	}
 
+	const declarationUrl = `/appeals/householder/submit/declaration?id=${journey.response.referenceId}`;
```

Comment:
Currently hardcoded for has submissions only, can be refactored when building the s78 submission journey?

If the questionnaire changes to using a declaration page as well this would need to be refactored for this as well